src/openrct2-ui/windows/Options.cpp

```diff
@@ -105,6 +105,7 @@ enum WINDOW_OPTIONS_WIDGET_IDX {
     WIDX_EFFECTS_GROUP,
     WIDX_DAY_NIGHT_CHECKBOX,
     WIDX_ENABLE_LIGHT_FX_CHECKBOX,
+    WIDX_ENABLE_LIGHT_FX_FOR_VEHICLES_CHECKBOX,
```

Comment:
Indentation looks incorrect

Reply:
It looks like 4 spaces to me. If there is an issue I can't see it.